test/e2e/testdata/acme/certificate_ibmcis.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: letsencrypt-cert
+spec:
+  dnsNames:
+    - RANDOM_STR.DNS_NAME
+    - '*.RANDOM_STR.DNS_NAME'
+  issuerRef:
+    name: letsencrypt-dns01-explicit
+    kind: ClusterIssuer
+  secretName: letsencrypt-cert
```

Comment:
the gcp testdata is presented in a different order. https://github.com/openshift/cert-manager-operator/blob/master/test/e2e/testdata/acme/certificate_gcp.yaml

I think we're OK...